frontend/tests/playwright/utils.ts

```diff
@@ -6,6 +6,7 @@ export const interceptLogin = async (page: Page, userParams?: Object) => {
       algoliaApiKeyProduct: null,
       username: 'john',
       unique_username: 'john123',
+      privileges: [],
```

Comment:
Wow that was all it took 😓.

Is there any way we can assign a matching type to this user object to keep it up-to-date? This property was not needed before to make the test pass.

Reply:
Let's make an issue for that. It's a small gotcha that took a bit to uncover.